checklists/alz_checklist.en.json

```diff
@@ -312,7 +309,7 @@
         {
             "category": "Identity and Access Management",
             "subcategory": "Identity",
-            "text": "Avoid using on-premises synced accounts for Microsoft Entra ID role assignments.",
+            "text": "Do not use on-premises synced accounts for Microsoft Entra ID role assignments.",
```

Comment:
Are there any edge cases where this is required? I know there were changes with action groups to allow group based notifications recently